src/app/component/result/result.component.ts

```diff
@@ -8,7 +8,7 @@ import quizData from 'src/assets/json/data.json';
   styleUrls: ['./result.component.css'],
 })
 export class ResultComponent implements OnInit, OnDestroy {
-  quizData: any = { ...quizData };
+  quizData = { ...quizData };
   isQuizCompleted: any;
   userName: any;
   points: any;
```

Comment:
Same with this

Reply:
correctAnswer: any;
  inCorrectAnswer: any; 
  
  Please remove this function

Reply:
Please remove questionservice

Reply:
i removed all unnecessary functions